cylc/flow/task_events_mgr.py

```diff
@@ -745,9 +786,8 @@ def process_message(
             )
 
         if message == self.EVENT_STARTED:
-            if (
-                flag == self.FLAG_RECEIVED
-                and itask.state.is_gt(TASK_STATUS_RUNNING)
+            if flag == self.FLAG_RECEIVED and itask.state.is_gt(
+                TASK_STATUS_RUNNING
```

Comment:
IMO we should avoid making purely style changes that aren't mandated by our agreed CI checks. Otherwise blocks like this might slowly oscillate between different developers personal preferences over time. In this case I think the original is more readable as the "state > running"  bit is not broken up. However, not a biggie, probably not worth changing back. (And I'm sure I'm guilty of the same thing sometimes 🤣 )

Reply:
Well we have a black/ruff config in our pyproject.toml. Although it isn't part of CI don't be suprised if autoformatting occurs that blackens/ruffens the code